src/global.d.ts

```diff
@@ -0,0 +1,18 @@
+interface Todo {
+  userId: number,
+  id: number,
+  title: string,
+  completed: boolean,
+  user: User | null,
+}
+
+interface TodoInfo {
+  title: string,
+  completed: boolean
+}
```

Comment:
https://www.typescriptlang.org/docs/handbook/utility-types.html
```suggestion
interface TodoInfo = Pick<Todo, "title" | "completed">
```